README.md

```diff
@@ -1,13 +1,19 @@
-# keri
+# KERI
 Key Event Receipt Infrastructure - the spec and implementation of the KERI protocol
 
+<img src="./images/KERI_image_by_CChunningham.png" alt="KERI image by CChunningham" border="0" width="600">
```

Comment:
I would recommend not mix markdown with html, you can use instead: ![](./images/KERI_image_by_CChunningham.png =600)

Reply:
Would it be possible to also give an atribute type=float:right ?

Reply:
I discovered GitHub markdown is a bit different than other implementations and haven't been able to figure out how to do it in strict markdown. 

 ![](./images/KERI_image_by_CChunningham.png =600)

won't work I am afraid, so I recovered the html version

Reply:
feel free to show how it's done, I am happy to learn and adopt!